packages/nimble-components/CONTRIBUTING.md

```diff
@@ -263,6 +263,16 @@ With an attribute defined there are several ways to react to updates. To minimiz
 
     Some valid use cases are reflecting correct aria values based on the updated attribute or forwarding updates to child components.
 
+#### Only throw exceptions in production for Nimble logic errors
```

Comment:
I welcome feedback on this. I tried to be assertive but may have overstepped.

I found one case that doesn't obey the guidelines as written: dialog and drawer both throw when the client calls `close` on something that's already closed (or open on something that's open). I'm open to feedback about whether we should change them to noop or update the guidelines to allow that class of exceptions. Maybe it's ok to throw from method calls but not from state updates?